`tests/test_executors.py`, `test_asyncio_progressbar_executor` is failing on darwin:
```
=================================== FAILURES ===================================
______________________ test_asyncio_progressbar_executor _______________________
    @pytest.mark.asyncio
    async def test_asyncio_progressbar_executor():
        tasks = [(func, [n], {}) for n in range(10)]
    
        executor = AsyncioProgressbarExecutor(logger=logger)
        # no guarantees for the results order
        assert sorted(await executor.run(tasks)) == [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
        assert executor.execution_time > 0.2
>       assert executor.execution_time < 0.3
E       assert 0.7072508335113525 < 0.3
E        +  where 0.7072508335113525 = <maigret.executors.AsyncioProgressbarExecutor object at 0x10fa2db70>.execution_time
tests/test_executors.py:37: AssertionError
```

I might need to dig deaper but this test seems to be dependant on time.

--- a/pkgs/tools/security/maigret/default.nix
+++ b/pkgs/tools/security/maigret/default.nix
@@ -1,19 +1,21 @@
 { lib
+, stdenv
 , fetchFromGitHub
 , python3
 }:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "maigret";
-  version = "0.4.3";
+  version = "0.4.4";
 
   src = fetchFromGitHub {
     owner = "soxoj";
     repo = pname;
     rev = "refs/tags/v${version}";
-    sha256 = "sha256-0Ni4rXVu3ZQyHBvD3IpV0i849CnumLj+n6/g4sMhHEs=";
+    hash = "sha256-Z8SnA7Z5+oKW0AOaNf+c/zR30lrPFmXaxxKkbnDXNNs=";
   };
 
+  nativeBuildInputs = [ python3.pkgs.pythonRelaxDepsHook ];
   propagatedBuildInputs = with python3.pkgs; [
     aiodns
     aiohttp
@@ -24,6 +26,7 @@ python3.pkgs.buildPythonApplication rec {
     beautifulsoup4
     certifi
     chardet
+    cloudscraper
     colorama
     future
     html5lib
@@ -60,14 +63,8 @@ python3.pkgs.buildPythonApplication rec {
     pytestCheckHook
   ];
 
-  postPatch = ''
-    # Remove all version pinning
-    sed -i -e "s/==[0-9.]*//" requirements.txt
-
-    # We are not build for Python < 3.7
-    substituteInPlace requirements.txt \
-      --replace "future-annotations" ""
-  '';
+  pythonRelaxDeps = true;
+  pythonRemoveDeps = [ "future-annotations" ];
 
   pytestFlagsArray = [
     # DeprecationWarning: There is no current event loop
@@ -82,6 +79,10 @@ python3.pkgs.buildPythonApplication rec {
     "test_pdf_report"
     "test_self_check_db_negative_enabled"
     "test_self_check_db_positive_enable"
+  ] ++ lib.optionals stdenv.isDarwin [
+    # AsyncioProgressbarExecutor is slower on darwin than it should be,
+    # Upstream issue: https://github.com/soxoj/maigret/issues/679
+    "test_asyncio_progressbar_executor"
   ];
 
   pythonImportsCheck = [

